app/api/groups/get_permissions_can_request_help_to.feature

```diff
@@ -55,5 +69,124 @@ Feature: Get permissions can_request_help_to for a group
+      | @Item | @Class                   | @Class            | item_unlocking   | false    | @AllUsers                         |                                                   |
+      | @Item | @ClassParentParent       | @Class            | item_unlocking   | false    | @HelperGroupNotVisible            | not visible                                       |
+      | @Item | @ClassParentParent       | @OtherSourceGroup | item_unlocking   | false    | @HelperOtherSourceGroup3          | other source group                                |
+      | @Item | @Class                   | @Class            | other            | false    | @AllUsers                         |                                                   |
+      | @Item | @ClassParentParent       | @Class            | other            | false    | @HelperGroupOther1                |                                                   |
+      | @Item | @ClassParent             | @Class            | other            | false    | @HelperGroupOther2                |                                                   |
+      | @Item | @ClassParent             | @OtherSourceGroup | other            | false    | @HelperOtherSourceGroup4          | other source group                                |
+      | @Item | @ClassParentParent       | @OtherSourceGroup | other            | false    | @HelperOtherSourceGroupNotVisible | other source group, not visible                   |
+  # The following lines are to make the groups visible by @Teacher
+  And the group @Teacher is a descendant of the group @HelperGroupSelf1
+  And the group @Teacher is a descendant of the group @HelperGroupGroupMembership1
+  And the group @Teacher is a descendant of the group @HelperGroupGroupMembership2
+  And the group @Teacher is a descendant of the group @HelperGroupItemUnlocking
+  And the group @Teacher is a descendant of the group @HelperGroupOther1
+  And the group @Teacher is a descendant of the group @HelperGroupOther2
+  And the group @Teacher is a descendant of the group @HelperOtherSourceGroup1
+  And the group @Teacher is a descendant of the group @HelperOtherSourceGroup2
+  And the group @Teacher is a descendant of the group @HelperOtherSourceGroup3
+  And the group @Teacher is a descendant of the group @HelperOtherSourceGroup4
+  When I send a GET request to "/groups/@Class/permissions/@Class/@Item"
+  Then the response code should be 200
+  And the response at $.granted_via_self.can_request_help_to[*] should be:
+    | id                       | name                          | is_all_users_group |
+    | @AllUsers                | AllUsers                      | true               |
+    | @HelperGroupSelf1        | Group HelperGroupSelf1        | false              |
+    | @HelperOtherSourceGroup1 | Group HelperOtherSourceGroup1 | false              |
```

Comment:
Just a remark: 
The response "checkers" (... should be) are usually JSON... even for an extract from json as here, it would make to have to have JSON. Is it on purpose to use a table instead? Typically, it is not clear (to me) if table may miss a column that we do not want to check.
I am fine with it but I was wondering it was on purpose.

Reply:
This method of checking doesn't necessary include all the fields that are present. It is made so that we check only what we need.

The column names are JsonPaths themselves. So it's possible to check for nested objects like `group.name`.

It suppress the need to update all old tests when we have change on other properties, but it can't check that all the properties are tested.